src/features/home/components/AttachedOfferCard.stories.tsx

```diff
@@ -1,18 +1,10 @@
-import { NavigationContainer } from '@react-navigation/native'
 import { ComponentStory, ComponentMeta } from '@storybook/react'
 import React from 'react'
 
 import { AttachedOfferCard } from './AttachedOfferCard'
 const meta: ComponentMeta<typeof AttachedOfferCard> = {
   title: 'ui/AttachedOfferCard',
   component: AttachedOfferCard,
-  decorators: [
```

Comment:
Il faut conserver le NavigationContainer pour Storybook :)

Reply:
Ah ! Inès le supprimait dans sa PR donc j'ai supposé que c'était inutile.

Reply:
Je fix !